bootstrap.html

```diff
@@ -26,5 +32,10 @@
       </div>
     </div>
   </div>
+
+  <!-- jQuery -->
+ <script src="js/jquery-latest.min.js"></script>
+<script src="js/bootstrap.js"></script>
+<script src="script.js/"></script>
 </body>
 </html>
```

Comment:
This wasn't taught anywhere, and it's a minor point, but whenever you create a new file, you should create a new line.